linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -129,6 +129,15 @@ MOCKABLE_FUNCTION(, THANDLE(THREADPOOL), threadpool_create, EXECUTION_ENGINE_HAN
 
 **SRS_THREADPOOL_LINUX_07_010: [** `insert_idx` and `consume_idx` for the task array shall be initialized to 0. **]**
 
+[Will be moved from threadpool_open]
+**S_R_S_THREADPOOL_LINUX_11_001: [** `threadpool_create` shall initialize internal threapool data items **]**
+
+[Will be moved from threadpool_open]
+**S_R_S_THREADPOOL_LINUX_07_020: [** `threadpool_create` shall create number of `min_thread_count` threads for `threadpool` using `ThreadAPI_Create`. **]**
+
+[Will be moved from threadpool_open]
+**S_R_S_THREADPOOL_LINUX_07_022: [** If one of the thread creation fails, `threadpool_create` shall fail and return a non-zero value, terminate all threads already created. **]**
```

Comment:
>terminate [](http://example.com/codeflow?start=134&length=9)

likely join is a better word, that's what our ThreadAPI abstraction is named.

Reply:
Okay.

Reply:
It should be Terminate, found a possible bug where it should be terminating but not terminating, instead waiting on join.